R/namespace_linter.R

```diff
@@ -0,0 +1,82 @@
+#' @describeIn linter that checks for missing packages and symbols in namespace calls
```

Comment:
Can we add a note here that this linter loads namespaces so it could potentially change the global state?

Reply:
Updated